.circleci/config.yml

```diff
@@ -64,17 +41,16 @@ workflows:
           filters:
             tags:
               ignore: /.*/
+      - dps/helm_lint
```

Comment:
I think `deploy_dev` should depend on this too

Reply:
good spot, fixed it